bin/assigner/example_data/call_block.json

```diff
@@ -32,7 +32,7 @@
             "receive_address" : "0x0000000000000000000000000000000000000003",
             "gas_price" : 3,
             "gas" : 200000,
-            "data" : "0x60006000600060006000600161007df1",
+            "data" : "0x60026003600060006000600161007df1",
```

Comment:
what exactly changed here?

Reply:
There change appeared to be non-functional, reverted, thanks